draft-ietf-ipsecme-rfc8229bis.xml

```diff
@@ -969,17 +1010,18 @@
           sending rate than the outer TCP would allow for.
           </t>
 
-          <t> TCP-in-TCP can also lead to “TCP meltdown”, where stacked instances 
-          of TCP can result in significant impacts to performance <xref target="TCP-MELTDOWN" />.
-          For the case in this document, such meltdown can occur when the window 
-          size of the outer TCP connection is smaller than the window sizes of the inner TCP
-          connections.  The resulting interactions can lead to packets backing up in the outer TCP
-          connection's send buffers.  In order to limit this effect, the outer
-          TCP connection should have limits on its send buffer size and on the
-          rate at which it reduces its window size.
+          <t> TCP-in-TCP can also lead to "TCP meltdown", where stacked instances
+          of TCP can result in significant impacts to performance
+          <xref target="TCP-MELTDOWN" />. This can occur when losses in the lower TCP (closer to the link)
+          Increase delays seen by the higher TCP (closer to the application) that create
```

Comment:
```suggestion
          increase delays seen by the higher TCP (closer to the application) that create
```

Reply:
Oh, this was in the Joe's text, I blindly copy-pasted it. Shame on me :-)